images/kubernetes-csi-livenessprobe/tests/01-help.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+if [[ "${IMAGE_NAME}" == "" ]]; then
+	echo "Must set IMAGE_NAME environment variable. Exiting."
+	exit 1
+fi
+
+docker run --rm "${IMAGE_NAME}" --help
```

Comment:
Let's figure out what we can test in this environment, and document manual testing steps for what else we can't test here.